### ISSUE TITLE
Biometric error

### ISSUE THREAD

**Issue by dado-vning** [PhilippC/keepass2android#1701]
Hello,
While trying to activate the biometric function, I get the following error :
![20210531_103037](https://user-images.githubusercontent.com/85107071/120192165-23797500-c21b-11eb-9da3-1b02364b351e.jpg)
 I have tried to unsinstall / re install version 1.085-r5 without solving

**Comment by felipeaf** [comment]
I also got an eissue here with biometry. I can enable quickUnlock but it is not asking ir fingerprints, but fallback to 3 characters from my password.  
I'm using sn Galaxy S20 FE, its bionetric sendor is under screen

**Comment by cpurules** [comment]
I also ran into this issue (up-to-date Galaxy S21, latest app version, also tried uninstalling/reinstalling).

I think the app is looking specifically for fingerprints, I only had face recognition on and was getting the error.  I added some fingerprints to test and then was able to set the app to unlock w/ fingerprint.

**Comment by milindrao** [comment]
Yes. I have the same problem on the Galaxy Tab S6 Lite which doesn't have a fingerprint reader and only the camera. Please fix this. Becomes very annoying to have to type in the password every time.

**Comment by PhilippC** [comment]
does it work in apps like Keepassdroid or KeepassDX?

**Comment by dado-vning** [comment]
Hi
I think I have solved my trouble. First my phone is rooted using magisk. Second, I was trying to succesfully pass the SafetyNet test in magisk, using the "universal safety net fix " module. In this case biometric error occured.
When I deactivate this module no more errors
At this time I am using the last Android 11 with last security updates.
